constrained-voucher.mkd

```diff
@@ -155,12 +155,20 @@ The term Registrar Voucher Request, or acronym RVR, is introduced to refer to th
 
 {{RFC8366}} provides for vouchers that assert proximity, that authenticate the Registrar and that can offer varying levels of anti-replay protection.
 
+The proximity proof provided for in {{RFC8366}}, is an assertion that the Pledge and the Registrar are believed to be in close together, from a network topology point of view.
+Like in {{RFC8955}}, proximity shown by making TLS connections between the Pledge and Registrar using IPv6 Link-Local addresses.
+
+The TLS connection is used to make a Voucher Request.
+This request is verified by the an agent of the Pledge's manufacturer, which then issues a voucher.
+The voucher provides an authorization statement from the manufacturer indicating that the Registrar is the intended owner of the device.
+The voucher refers to the Registrar through pinning of the Registrar's key.
```

Comment:
This reads a bit too specific: it may pin also the Registrar's cert, or the Domain CA cert.
key -> public key ?